api/hrlh.rb

```diff
@@ -10,6 +11,14 @@ class API < Grape::API
       def database
         @database ||= Orchestrate::Application.new(ENV['ORCH_API_KEY'],ENV['ORCH_REGION'])
       end
+
+      def interview_db
+        database['interviewer']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.